app/src/main/java/org/xbmc/kore/jsonrpc/type/ListType.java

```diff
@@ -284,6 +284,17 @@ public ItemBase(JsonNode node) {
             writer = JsonUtils.stringListFromJsonNode(node, WRITER);
             year = JsonUtils.intFromJsonNode(node, YEAR, -1);
         }
+
+        @Override
+        public boolean equals(Object obj) {
+            if (obj == this)
+                return true;
+
+            if (! (obj instanceof ItemBase))
+                return false;
+
+            return ((ItemBase) obj).id == this.id;
```

Comment:
This doesn't feel totally right.
What guarantees do we have that this id is unique? If we have them (or if we don't have them but are assuming it), we should explicitly make a comment documenting that.

Reply:
Code has been removed as it is not needed anymore.